tests/test_predictors/test_figure_table_predictors.py

```diff
@@ -119,3 +127,147 @@ def test_predict(self):
             {'id': 1, 'metadata': {}, 'spans': [{'end': 22214, 'start': 22042}]},
             {'id': 2, 'metadata': {}, 'spans': [{'end': 23502, 'start': 23400}]},
             {'id': 3, 'metadata': {}, 'spans': [{'end': 29584, 'start': 29369}]}]
+
+    def test_predict_2(self):
```

Comment:
I think a docstring on both tests or more exact name explaining what they're testing or confirming would be helpful